plugins/org.jkiss.dbeaver.ext.clickhouse/src/org/jkiss/dbeaver/ext/clickhouse/model/jdbc/ClickhouseResultSetMetaDataImpl.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2022 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.ext.clickhouse.model.jdbc;
+
+import org.jkiss.dbeaver.model.exec.jdbc.JDBCResultSet;
+import org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCResultSetMetaDataImpl;
+
+import java.sql.SQLException;
+
+/**
+ * JDBCUtils#normalizeIdentifier method usually use dor columns/tables name reading.
+ * normalizeIdentifier trim names, but we can create tables/columns with spaces around their names
+ * Let's just read names from the driver directly
```

Comment:
As I understand extra spaces around table/column names is quite specific for several databases. Why do you create code without trimming only for clickhouse? Could we make trimming database-specific and remove trimming from common code?

Reply:
Because I'm not sure, that only Firebird adds extra spaces around tables/columns names in the ResultSet. And we have banch of genric databases. I think, that tsis is very risky - do not trim result by default. 
Also I thinks, that spaces around tables/columns names - this is side case (and not the best idea).

Reply:
Okay, thanks